cea/optimization/master/master_to_slave.py

```diff
@@ -255,6 +256,7 @@ def calc_master_to_slave_variables(locator, gen,
 
     # initialise class storing dynamic variables transferred from master to slave optimization
     master_to_slave_vars = slave_data.SlaveData()
+    master_to_slave_vars.debug = config.general.debug
```

Comment:
what is the use of this?

Reply:
This is handed from 'evaluation.py' down to 'storage_tank_pcm.py' (through 'cooling_main.py', where the PCM storage is initialised). In 'storage_tank_pcm.py' you can see that the boolean is used to output basic properties of the storage upon initialisation and then charge-levels at each charging and discharging stage within 'cooling_resoucre_activation.py'.

The alternative would be to hand down config as a separate variable or to remove the debugging functionalities entirely. But I believe the current solution makes sense.

Reply:
thanks for the clarification!